test/integration/site_layout_test.rb

```diff
@@ -3,13 +3,13 @@
 class SiteLayoutTest < ActionDispatch::IntegrationTest
 
   def setup
-    @user = users(:ohm)
+    @user = users(:users_001)
   end
 
   test 'layout links' do
     get login_path
 
-    assert_select 'a[href=?]', help_path
+    #assert_select 'a[href=?]', help_path
```

Comment:
It doesn't really matter what's *on* the Help page (as of yet) - this test is supposed to verify that the links in the navbar and footer are showing up when you hit the login page and you're not logged in. Looking at [_header.html.erb](https://github.com/Ohmbrewer/ohmbrewer/blob/master/app/views/layouts/_header.html.erb), it seems like the desired functionality is that when you aren't logged in, the Help link doesn't show up.

Move that check down underneath the assert_template call and add a check for the home_path there as well.